packages/node/package.json

```
@@ -30,11 +30,11 @@
     "typescript": "^4.0.5"
   },
   "dependencies": {
+    "@inrupt/jose-legacy-modules": "*",
```

Why would we make this a wildcard dependency? It's just republished whenever `jose` is with a matching version number, so we can keep it up to date using Dependabot just like we do today.

```suggestion
    "@inrupt/jose-legacy-modules": "0.0.3-3.14.3",
```

(I suppose using the `^` could get us to suddenly match a major version bump, so pinning it to a specific version might be a good idea. Alternatively, we could rework jose-legacy-modules to switch the versions around, i.e. start with the jose version and then append the fork's version, since there's not going to be breaking changes in what the fork does anyway (I think).

---

Oops, it wasn't intentional, I wonder if that's NPM default behaviour for dependencies using unexpected version numbers format.